vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -156,22 +157,12 @@ func validateCreateOptions(r volume.Request, fsMap map[string]string) error {
 		r.Options[fsTypeTag] = fs.FstypeDefault
```

Comment:
I have a slightly different opinion here. `fs` is only managing the filesystem, and it could export the `FstypeDefault` constant to be used by other modules. Setting the create request options inside `fs` would introduce some coupling i.e. the `fs` module will now also have to know about volume requests.

As such, `VerifyFSSupport(fstype string)` only takes a `string` as an input, and the implementation knows nothing about volume requests.

Thoughts?